mmv1/products/redis/api.yaml

```diff
@@ -400,3 +400,8 @@ objects:
           an existing instance. For DIRECT_PEERING mode value must be a CIDR range of size /28, or
           "auto". For PRIVATE_SERVICE_ACCESS mode value must be the name of an allocated address 
           range associated with the private service access connection, or "auto".
+      - !ruby/object:Api::Type::String
+        name: customerManagedKey
```

Comment:
Can this field be updated?
If not, we probably want to add `input: true` to tell terraform to delete and recreate the resource if user's attempting to update the field.

Reply:
Done